content/docs/start/visualization.md

```diff
@@ -0,0 +1,21 @@
+# Visualization with DVC Plots
+
+In parallel with versioning parameters and metrics by associating them with
+experiments, DVC allows to generate plots from multivariate outputs. There are
+several options to generate and keep track of the plots:
+
+- DVC can generate HTML files that includes interactive Vega-Lite plots, from
+  the data series in JSON, YAML, CSV, or TSV.
+
+- DVC can keep track of image files produced as plot outputs from the
+  training/evaluation scripts.
+
+- DVC, with its DVCLive integration can produce plots automatically during
+  training.
+
+In this document, we'll focus on the first option and briefly describe the other
```

Comment:
dvclove is the third option though?

Reply:
What would you recommend on this? These three options look mutually exclusive to me, and structuring the document on one of them makes the others a bit distracting. If users are expected to learn the manual visualization, starting from DVCLive makes most of the stuff hidden _under the carpet._ If we start from manual plotting, telling DVCLive later will be confusing. 

I plan to start from generating a confusion matrix in Vega, then building a "misclassifed examples from each class" image. It will be like a confusion matrix but will show (28x28 image) samples instead of numbers. This will show the `dvc plots`' image capabilities. 

In this scenario, I think DVCLive can also be used to track the progress of training, but it may be distracting. WDYT? @shcheklein @dberenbaum